tests/foreman/ui/test_subscription.py

```diff
@@ -258,7 +260,7 @@ def test_positive_access_manifest_as_another_admin_user(test_name, target_sat):
     ).create()
     # use the first admin to upload a manifest
     with Session(test_name, user=user1.login, password=user1_password) as session:
-        target_sat.upload_manifest(org.id)
+        target_sat.upload_manifest(org.id, function_entitlement_manifest.content)
```

Comment:
We could have used `function_entitlement_manifest_org` instead ?

Reply:
No, as the purpose of this test is to validate that users other than `admin` can manage manifests, and `function_entitement_manifest_org` uploads the manifest as  the `admin` user.